app/services/moab_processing_service.rb

```diff
@@ -8,28 +8,47 @@ def self.process(druid:, force: false)
 
   # @param [String] druid
   # @param [Boolean] force even if md5 match
-  def initialize(druid:, force: false)
+  # @param [Symbol] queue
+  def initialize(druid:, force: false, queue: :retro)
```

Comment:
you want the default to be :retro??

Reply:
Yes, since that is the lowest priority queue. In the future, might make sense to change this, but for now is reasonable.